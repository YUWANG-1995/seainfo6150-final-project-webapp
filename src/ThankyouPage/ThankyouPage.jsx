import React from "react";
import styles from "./ThankyouPage.module.css";

const ThankyouPage = () => {
  return (
    <div className={styles.container}>
      <p className={styles.text}> Successful! we already received your  </p>
      <p className={styles.text}> we already received your oreder information !</p>
      <p className={styles.text}> we will contact you soon !</p>
    </div>
  )
}

export default ThankyouPage;
