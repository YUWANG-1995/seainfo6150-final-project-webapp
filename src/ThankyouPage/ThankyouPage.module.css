.container{
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  margin-top: 20px;
  padding-top: 30px;
}

.text{
  text-align: center;
  font-family: 'Roboto', sans-serif;
  font-weight: bold;
  font-size: 1.5em;
  color: #f5222d;
}
