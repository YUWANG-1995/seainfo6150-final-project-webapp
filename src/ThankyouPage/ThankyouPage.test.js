import React from "react";
import { render } from "@testing-library/react";
import ThankyouPage from "./ThankyouPage.jsx";

it("renders the ThankyouPage Component Correctly", () => {
  const { container } = render(<ThankyouPage />);
  expect(container).toMatchSnapshot();
});
