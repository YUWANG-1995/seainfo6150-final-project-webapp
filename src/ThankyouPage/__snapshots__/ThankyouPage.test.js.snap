// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the AboutUs Component Correctly 1`] = `
<div>
  <div
    class="container"
  >
    <p
      class="text"
    >
       Successful! we already received your  
    </p>
    <p
      class="text"
    >
       we already received your oreder information !
    </p>
    <p
      class="text"
    >
       we will contact you soon !
    </p>
  </div>
</div>
`;

exports[`renders the ThankyouPage Component Correctly 1`] = `
<div>
  <div
    class="container"
  >
    <p
      class="text"
    >
       Successful! we already received your  
    </p>
    <p
      class="text"
    >
       we already received your oreder information !
    </p>
    <p
      class="text"
    >
       we will contact you soon !
    </p>
  </div>
</div>
`;
